lib/pipes/parseBoolean.pipe.ts

```diff
@@ -1,13 +1,20 @@
 import { BadRequestException } from '../exceptions';
+import type { ParameterPipe, PipeOptions } from './ParameterPipe';
 
-export function ParseBooleanPipe(value: any): boolean {
-  if (value === true || value === 'true') {
-    return true;
-  }
+export function ParseBooleanPipe(options?: PipeOptions): ParameterPipe<boolean> {
+  return (value: any, name?: string) => {
+    if (!options?.nullable && value == null) {
```

Comment:
Would abstract this `if` statement into a helper function (since all pipes will have it), or we might need to use Classes instead at some point so we can extend the `BasePipe`